website/docs/connect-data/how-to-guides/setup-datasource-environments.md

```diff
@@ -1,9 +1,20 @@
- # Setup Multiple Datasource Environments
-
-:::info
-The Datasource environments feature is only available on [Business Edition](https://www.appsmith.com/pricing). This feature is still in Beta. If you have questions and would like to test this feature and share feedback, please contact the support team using the chat widget at the bottom right of this page. 
-:::
-
+---
+title: Setup Multiple Datasource Environments
```

Comment:
<img width="1069" alt="Screenshot 2023-09-21 at 21 50 04" src="https://github.com/appsmithorg/appsmith-docs/assets/32227135/5747a760-288f-4b8e-b893-748408685b39">

looks odd on this page

Reply:
also this is out of beta right?

Reply:
Not sure. Pranay Agrawal will confirm. For now, I have replicated the call-outs into tags.

Reply:
We can remove the beta tag once confirmed - @pranayagarwal96 - Please confirm this.